### ISSUE TITLE
Content release in tugboat does not build with other FE branches

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#4966]
**Describe the defect**
When I am in the CMS on a tugboat or demo environment and I select a different front end branch / PR to use,  the web version is not built, despite often showing success in the interface.

![image](https://user-images.githubusercontent.com/5752113/113927784-e913ce80-97bb-11eb-85ca-bc23d422ab68.png)

![image](https://user-images.githubusercontent.com/5752113/113927793-ee711900-97bb-11eb-8dcb-ed6eb91020cd.png)

![image](https://user-images.githubusercontent.com/5752113/113927805-f29d3680-97bb-11eb-9d32-4402e126db67.png)

from slack here https://dsva.slack.com/archives/CT4GZBM8F/p1617822749421700

**To Reproduce**
1. Go to '/admin/content/deploy'
2. Click on 'Select a different frontend branch' and choose an existing branch or PR
3. click 'Release content '
4. Watch it progress util the status reports success (if it reports failure, pick a different PR as it may just be that PR)
5. Click the "Go to front end" button. 
6. See page `/admin/content/deploy`

**Expected behavior**
- [ ] When the status says `success`, the web site is actually there.
- [ ] When the build fails to complete, it says Error.
- [ ] When a PR  or branch is selected it is used to successfully build the web. 

**Screenshots**

**Desktop (please complete the following information if relevant, or delete):**

**Additional context**
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS objective (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

**Comment by ElijahLynn** [comment]
Do you have more of the text output for the false success you can paste?

**Comment by ndouglas** [comment]
<img width="442" alt="Screen Shot 2021-04-08 at 10 07 10 AM" src="https://user-images.githubusercontent.com/1318579/114041170-4b1f1300-9852-11eb-8460-323b2d2a8f14.png">

I hypothesize that this is a result of the `yarn build` command not returning a correct (i.e. error) exit code in this (and possibly other) cases, since the `echo 'Replace s3 address with local in generated files'` command is executed.  If you insert a `false` in the composer command, it's interpreted correctly and execution halts at that point with an error, as expected:

<img width="649" alt="Screen Shot 2021-04-08 at 10 14 26 AM" src="https://user-images.githubusercontent.com/1318579/114042253-46a72a00-9853-11eb-9f14-51dfb0ac0f14.png">

I introduced intentional build errors while testing the work that I did following up the work done by @acrollet and @indytechcook and errors were recognized and displayed to the user appropriately.  Indeed, I recall seeing a number of errors 😂

**Comment by ndouglas** [comment]
[the full text](https://gist.github.com/ndouglas/ad94fe4c877d790f49a9f34ac03b88f1) of one of these failed builds, for the record

**Comment by indytechcook** [comment]
> the full text of one of these failed builds, for the record

THis is not a failure.  it's just log messages from webpack.

**Comment by indytechcook** [comment]
I'm not sure this is really an error.  What you are seeing in the logs are 2 different steps.  Step 1 is running webpack for the new branch and step 2 is running the content build

each command would be a separate entry in watchdog

**Comment by indytechcook** [comment]
This issue is related to the build script.  We are hitting this case:

```
# Build the content
if [ "${buildtype}" != "vagovdev" ]
then
    yarn build:content $args
fi

```

**Comment by indytechcook** [comment]
Related Pr which caused our issue: https://github.com/department-of-veterans-affairs/vets-website/pull/16631

**Comment by indytechcook** [comment]
PR to `vets-website` repo: https://github.com/department-of-veterans-affairs/vets-website/pull/16726

**Comment by indytechcook** [comment]
PR to use new flag which should fix the issue: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/5013

**Comment by swirtSJW** [comment]
I just merged #5013.  Hopefully that resolves the issue of the failed Web builds on tugboat.   

I created this issue though https://github.com/department-of-veterans-affairs/va.gov-cms/issues/5032 as I think it is a bad user experience to be told there is a success when Web was not available.

**Comment by indytechcook** [comment]
We could also use our own build script which wouldn't rely on another repo.
